tests/functional/aws-node-sdk/test/bucket/putBucketNotification.js

```diff
@@ -169,17 +193,21 @@ describe('aws-sdk test put notification configuration', () => {
         ];
 
         corsTests.forEach(test => {
-            it(`should ${test.it}`, done => {
-                const req = s3.putBucketNotificationConfiguration(test.param);
-                req.httpRequest.headers.origin = 'http://localhost:3000';
-                req.send(err => {
+            it(`should ${test.it}`, async () => {
+                // For v3, we need to modify the request through middleware instead of .httpRequest.headers
```

Comment:
```suggestion
                // With AWS SDK v3, we need to modify the request through middleware instead of .httpRequest.headers
```

just because later the comment might be hard to understand without context on when the lines were added